weather_mv/loader_pipeline/util_test.py

```diff
@@ -178,5 +185,99 @@ def test_make_attrs_ee_compatible_a2(self):
 
 
 class ToJsonSerializableTypeTests(unittest.TestCase):
-    # TODO(#106): Write tests...
-    pass
+
+    def test_to_json_serializable_type_none(self):
+        attrs = None
+
+        expected = None
+
+        actual = to_json_serializable_type(attrs)
+
+        self.assertEqual(actual, expected)
+        self.assertEqual(type(actual), type(expected))
+
+    def test_to_json_serializable_type_npfloat(self):
+        attrs = np.float32(1)
+
+        expected = float(1)
+
+        actual = to_json_serializable_type(attrs)
```

Comment:
Similar to my suggestion above. To help you out, here are a bunch of cases to check: 

```suggestion
    self.assertIsInstance(self.convert(0.1), float)
    self.assertIsInstance(self.convert(np.float32('0.1')), float)
    self.assertIsInstance(self.convert(np.float32('1')), float)
    self.assertIsInstance(self.convert(np.float16('0.1')), float)
    self.assertIsInstance(self.convert(np.single('0.1')), float)
    self.assertIsInstance(self.convert(np.double('0.1')), float)
    self.assertNotIsInstance(self.convert(1), float)
    self.assertNotIsInstance(self.convert(np.csingle('0.1')), float)
    self.assertNotIsInstance(self.convert(np.cdouble('0.1')), float)
    self.assertNotIsInstance(self.convert(np.intc('1')), float)
```

Reply:
Thanks for these cases. Added in https://github.com/google/weather-tools/pull/327/commits/d9cf97add436e462d01fb43fe2d32ac7c0511d17.